packages/go_router_builder/example/lib/main.dart

```diff
@@ -35,6 +35,7 @@ class App extends StatelessWidget {
   late final GoRouter _router = GoRouter(
```

Comment:
Since you have already create the another shellroute example let's revert the change to the main.dart. As a main example, I think we should not make it more complex whenever we add new feature

Reply:
I reverted the changes here: [548882a](https://github.com/flutter/packages/pull/3439/commits/548882ac28c7196aa286f05384bec1c4c34edfcf)